contracts/Vault.vy

```diff
@@ -1536,6 +1552,7 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     # Update reporting time
     self.strategies[msg.sender].lastReport = block.timestamp
     self.lastReport = block.timestamp
+    self.lockedProfit = gain # profit is locked and gradually released per block
```

Comment:
i was trying to see if there was a way to fix this from baseStrategy, but realized is going to complicate the strat code on the reporting, also is like "lying" to the vault about the actual gain, i think is the concern of the vault to keep accurate share price so your fix needs to be on the Vault